client/scss/components/_image_gallery.scss

```diff
@@ -3,6 +3,8 @@
   width: 100%;
   display: flex;
   margin-bottom: $vertical-space-unit;
+  overflow: scroll;
+  -webkit-overflow-scrolling: touch;
```

Comment:
I can't remember what we decided about the `touch-scroll` class in `_root_scope_classes.scss` – it adds this property, in any event.